try_premium_at_start_old_account_older_remote_ts failed in Travis

## Problem Definition

Failed with:

```
_________ test_try_premium_at_start_old_account_older_remote_ts[True] __________
rotkehlchen_instance = <rotkehlchen.rotkehlchen.Rotkehlchen object at 0x7f3d15989550>
username = 'testuser', db_password = '123'
rotki_premium_credentials = <rotkehlchen.premium.premium.PremiumCredentials object at 0x7f3d15dbd2b0>
    @pytest.mark.parametrize('start_with_valid_premium', [True])
    def test_try_premium_at_start_old_account_older_remote_ts(
            rotkehlchen_instance,
            username,
            db_password,
            rotki_premium_credentials,
    ):
        setup_starting_environment(
            rotkehlchen_instance=rotkehlchen_instance,
            username=username,
            db_password=db_password,
            premium_credentials=rotki_premium_credentials,
            first_time=False,
            same_hash_with_remote=False,
            newer_remote_db=False,
            db_can_sync_setting=True,
        )
        # DB should not have changed
>       assert rotkehlchen_instance.data.db.get_main_currency() == DEFAULT_TESTS_MAIN_CURRENCY
E       AssertionError: assert <Asset identi...nd symbol:GBP> == <Asset identi...ro symbol:EUR>
E         Omitting 7 identical items, use -vv to show
E         Differing attributes:
E         identifier: 'GBP' != 'EUR'
E         name: 'Great British Pound' != 'Euro'
E         symbol: 'GBP' != 'EUR'
E         started: None != 915148800
```

https://travis-ci.com/github/rotki/rotki/jobs/395082075#L1123

## Task

Find out why and fix it if not flaky. IF flaky fix the flakiness

--- a/rotkehlchen/tests/integration/test_premium.py
+++ b/rotkehlchen/tests/integration/test_premium.py
@@ -298,8 +298,7 @@ def test_try_premium_at_start_old_account_older_remote_ts(
         newer_remote_db=False,
         db_can_sync_setting=True,
     )
-    # DB should not have changed
-    assert rotkehlchen_instance.data.db.get_main_currency() == DEFAULT_TESTS_MAIN_CURRENCY
+    assert_db_got_replaced(rotkehlchen_instance=rotkehlchen_instance, username=username)
 
 
 @pytest.mark.parametrize('start_with_valid_premium', [True])

